### ISSUE TITLE
Unable to replicate the sample results

### ISSUE THREAD

**Issue by anilsathyan7** [limingcv/Photorealistic-Style-Transfer#10]
Hi,
     I ran the training script(IPython in Google Colab) for 30,000 iterations; but was unable to get the same results (as given in the sample). Ideally how many epochs should we run it to achieve the same result.What would be the total loss in that scenario?

Have you trained with larger input size (or a different parameter setting)or applied any other post-processing?

Here is my result after 30k iterations 
![myout](https://user-images.githubusercontent.com/1130185/65820394-d6731d80-e245-11e9-9c16-c2b6ed423528.png)

Here is your output:-

![ideal_output](https://user-images.githubusercontent.com/1130185/65820408-0fab8d80-e246-11e9-9aa8-004f624c234c.png)
